CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 
+### Compatibility issues
+
+* Removing the deprecated awsx.Cluster and awsx.Network type (deprecated in 0.18.6). Code that uses
```

Comment:
Does this warrant a minor version bump (rather than a patch version bump)? Are there more deprecated types that we plan to remove?

Reply:
@pgavlin: 

> Note: i'm putting this into a 1.0 branch as this is a breaking change

:)

Reply:
Note that I think we’ll want to release this in our next minor version bump, but will likely want to do that before 1.0. 

We don’t want 0.x to 1.0 to be a meaningful breaking change, it should be more like the end of a 1.0-beta-x chain.

Reply:
Gotcha.  So should this be an 0.19.x change then?  Based on our compat guidelines, it should be, but i just want to make sure that we're on the same page here.